.github/workflows/release-please-app.yml

```diff
@@ -4,7 +4,7 @@ name: Release new app version
 on:
   push:
     branches:
-      - "master"
+      - "main"
```

Comment:
Why does this one use quotes?

Reply:
Sounds good. We should probably harmonize across workflows (or use a linter / formatter).